src/firefly/js/visualize/PlotViewUtil.js

```diff
@@ -63,6 +63,11 @@ export function getPlotViewIdxById(visRoot,plotId) {
     return visRoot.plotViewAry.findIndex( (pv) => pv.plotId===plotId);
 }
 
+export function getPlotGroupIdxById(visRoot,plotGroupId) {
+    if (!plotGroupId) return null;
+    return visRoot.plotGroupAry.findIndex( (pg) => pg.plotGroupId===plotGroupId);
+}
+
```

Comment:
This is good but I have been trying to make the utilities more flexibile. Instead of just visRoot it should take visRoot or plotGroupAry.
_something like:_

```
export function getPlotGroupIdxById(ref,plotGroupId) {
    if (!ref) return null;
    const plotGroupAry= ref.plotGroupAry ? ref.plotGroupAry : ref;  // this line determine what was passed
    return plotGroupAry.findIndex( (pg) => pg.plotGroupId===plotGroupId);
}
```